data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodeslegend.asset

```diff
@@ -0,0 +1,35 @@
+local TexturesPath = asset.syncedResource({
+    Type = "HttpSynchronization",
+    Name = "Streamnodes textures",
+    Identifier = "streamnodes_legend",
+    Version = 1
+})
+
+asset.require("spice/base")
+local assetHelper = asset.require("util/asset_helper")
+--local textures = asset.require('./streamnodes_textures').TexturesPath
+
+local legend = {
+    Identifier = "Legendfluxnodes",	
```

Comment:
extra tab at the end of this line 

(make sure to remove tabs altogether in this file. I recommend using a tool to visualize them in the editor)